### DISCUSSION TITLE
Cannot find console merge feature for Microfrontend Composer pages

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#352] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v12.0.0

### Description

Hi, I can't find anymore the merge feature for the Microfrontend Composer pages.
Am I doing something wrong? Does it happen to you?

Here in the doc the section is found between 'Authorization' and 'Fast Data': https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/merge_collaboration#how-to-perform-a-merge-of-configurations

And here is my screenshot (between the two branches there are 8 new pages):
![image](https://github.com/mia-platform/community/assets/39826608/71850467-57d8-454e-9e38-0885ac65ffaa)

### Actual Outcome

I cannot merge Microfrontend files

### Expected Outcome

I can merge Microfrontend files

**Comment by fredmaggiowski** [comment]
Hey, the microfrontend composer does not support the merge, yet

@TheRusCompass fyi

**Reply by edocrippaofficial** [reply]
Wait, I'm sure I've already done it a lot of times in the past.

Or it's a giant Mandela effect 😄

**Reply by fredmaggiowski** [reply]
yes, it was available but has been temporarely disabled

**Reply by edocrippaofficial** [reply]
Understood.

Is is stated somewhere? If I hadn't figured this out I would have lost a whole day's work...  
I suggest to also include this information in the merge dialog, because it's super easy to miss/forget

**Reply by fredmaggiowski** [reply]
@TheRusCompass isn't this stated in a release note? I agree that we should provide this information

**Reply by theliuk** [reply]
And what is the user experience of a poor daemon who has to work with BO?

**Reply by TheRusCompass** [reply]
we are currently investigating why it is not appearing anymore in the merging modal. It is an unwanted behavior. We will let you know as soon as it is fixed.

**Reply by TheRusCompass** [reply, marked as answer]
Hello, this fix is available in the v12.0.2 that has already been released in Preview!

**Comment by theliuk** [comment]
I could swear there was, I used it multiple times and I also opened a ticket about the difficulty to merge BO pages via the UI tool of the console.
